addons/survey/models/survey_question.py

```diff
@@ -264,15 +264,8 @@ def _compute_question_ids(self):
         and lower than the index of the next page."""
         for question in self:
             if question.is_page:
-                next_page_index = False
-                for page in question.survey_id.page_ids:
-                    if page._index() > question._index():
-                        next_page_index = page._index()
-                        break
-
-                question.question_ids = question.survey_id.question_ids.filtered(
-                    lambda q: q._index() > question._index() and (not next_page_index or q._index() < next_page_index)
-                )
+                question.question_ids = question.survey_id.question_ids\
```

Comment:
Note that method docstring is somehow outdated (and can probably be simply removed)

Reply:
bien vu ;) updated